front/lib/models.ts

```diff
@@ -471,6 +482,8 @@ export class DataSource extends Model<
   declare visibility: "public" | "private";
   declare config?: string;
   declare dustAPIProjectId: string;
+  declare connectorId?: string;
+  declare connectorProvider?: "slack" | "notion" | "google";
```

Comment:
Can we use `ConnectorProvider` `@connectors/types` ?

Reply:
I tried but it did not compile :/

Reply:
Yeah makes sense let's try to not import anything then